Resources/public/js/views/services/ez-contentcreateviewservice.js

```diff
@@ -93,6 +107,48 @@ YUI.add('ez-contentcreateviewservice', function (Y) {
                 });
             });
         },
+
+        /**
+         * changeLanguage event handler. It opens languageSelectionBox for selecting
+         * language of created content
+         *
+         * @method _changeLanguage
+         * @private
+         * @param {EventFacade} e
+         */
+        _changeLanguage: function (e) {
+            var that = this;
+
+            e.preventDefault();
+            this.fire('languageSelect', {
+                config: {
+                    title: "Change language to:",
+                    languageSelectedHandler: Y.bind(this._changeContentLanguage, this),
+                    cancelLanguageSelectionHandler: null,
+                    canBaseTranslation: false,
+                    newTranslationsOnList: true,
+                    existingTranslations: [that.get('languageCode')]
+                },
+            });
+        },
+
+        /**
+         * Changes language od created content to one given in event facade
+         *
+         * @method _changeContentLanguage
+         * @private
+         * @param {EventFacade} e
+         * @param {String} e.selectedLanguageCode language code to which created ontent will be switched
+         */
+        _changeContentLanguage: function (e) {
+            var version = this.get('version'),
+                service = this;
+
+            version.sync('delete', {api: this.get('capi')}, function () {
```

Comment:
even if `sync` is pubic, you are not really supposed to call it directly. You should use [`Model.destroy`](http://yuilibrary.com/yui/docs/api/classes/Model.html#method_destroy) with the `remove` option instead.